### DISCUSSION TITLE
No Hello World... but a start all the same.

### DISCUSSION THREAD

**Discussion by WolfStar76** [skills/.github#259] [General]
It's interesting how, when you haven't done any coding since the mid-80's (Apple Basic, yo!) - how even simple things change.

It used to be that a simple "Hello World" was how you demonstrated that you understood the VERY basics of your new programming language.

Now it's all about branches, commits, and pulls. 

But... at least I got to make a first pass at a profile. 

A decent start to the basics. :)

**Comment by sinsukehlab** [comment]
Git is just a version control system. You still start with "Hello World" when you learn a new programming language. You might want to recall specific versions later while writing codes. Git records changes to a file or set of files over time so that you can recall specific versions later.

**Comment by LUCCASENN** [comment]
How did i use that for????
I did like for someone help me -PLEASE

**Reply by sinsukehlab** [reply]
Please open a new discussion (Q&A) and describe what you would like to do.
